config/grpc_client_gen.config

```diff
@@ -1,7 +1,7 @@
 {plugins, [
     {grpcbox_plugin,
         {git, "https://github.com/andymck/grpcbox_plugin.git",
```

Comment:
Should we bring this into github.com/helium ?

Reply:
Not at this point.  This is a fork of an external lib and I am hoping my changes will be merged into the original

Reply:
👍🏻